data_steward/tools/purge_hpo_data.py

```diff
@@ -0,0 +1,100 @@
+"""
```

Comment:
you mentioned not loving the module name.  I'll list some options. 
- purge_site_data
- exclude_site_submission
- exclude_site_submission_data

 I _think_ I like `exclude_site_submission` because it seems like we are only interested in dropping the info a site submitted (instead of non-CDM tables).   This is purely optional.  I just wanted to offer some ideas.

Reply:
Just renamed it to exclude_site_submission